### ISSUE TITLE
`cargo install` failed

### ISSUE THREAD

**Issue by marcospb19** [Wilfred/difftastic#245]
## Problem

Ran `cargo install difftastic` and it failed.

`cargo` output:

<details>
  <summary>Click to expand.</summary>
  
  ```txt
   Compiling autocfg v1.1.0
   Compiling libc v0.2.122
   Compiling cc v1.0.73
   Compiling memchr v2.4.1
   Compiling crossbeam-utils v0.8.8
   Compiling lazy_static v1.4.0
   Compiling cfg-if v1.0.0
   Compiling scopeguard v1.1.0
   Compiling rayon-core v1.9.1
   Compiling proc-macro2 v1.0.37
   Compiling log v0.4.16
   Compiling unicode-xid v0.2.2
   Compiling regex-syntax v0.6.25
   Compiling quick-error v1.2.3
   Compiling either v1.6.1
   Compiling termcolor v1.1.3
   Compiling static_assertions v1.1.0
   Compiling hashbrown v0.11.2
   Compiling textwrap v0.15.0
   Compiling strsim v0.10.0
   Compiling same-file v1.0.6
   Compiling bitflags v1.3.2
   Compiling owo-colors v3.3.0
   Compiling typed-arena v2.0.1
   Compiling radix-heap v0.4.2
   Compiling rustc-hash v1.1.0
   Compiling wu-diff v0.1.2
   Compiling humantime v1.3.0
   Compiling memoffset v0.6.5
   Compiling crossbeam-epoch v0.9.8
   Compiling rayon v1.5.1
   Compiling indexmap v1.8.1
   Compiling archery v0.4.0
   Compiling walkdir v2.3.2
   Compiling itertools v0.10.3
   Compiling libmimalloc-sys v0.1.24
   Compiling tree-sitter v0.20.6
   Compiling rpds v0.10.0
   Compiling crossbeam-channel v0.5.4
   Compiling aho-corasick v0.7.18
   Compiling os_str_bytes v6.0.0
   Compiling atty v0.2.14
   Compiling term_size v0.3.2
   Compiling num_cpus v1.13.1
   Compiling quote v1.0.17
   Compiling regex v1.5.5
   Compiling crossbeam-deque v0.8.1
   Compiling const_format_proc_macros v0.2.22
   Compiling clap v3.1.8
   Compiling env_logger v0.7.1
   Compiling const_format v0.2.22
   Compiling mimalloc v0.1.28
   Compiling pretty_env_logger v0.4.0
   Compiling difftastic v0.25.0
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-m64" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.0.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.1.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.10.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.11.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.12.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.13.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.14.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.15.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.2.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.3.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.4.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.5.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.6.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.7.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.8.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.difft.71f95a22-cgu.9.rcgu.o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361.4um47g0dgyomkzby.rcgu.o" "-Wl,--as-needed" "-L" "/tmp/cargo-installEL3oso/release/deps" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out" "-L" "/tmp/cargo-installEL3oso/release/build/libmimalloc-sys-8f98f989a6e65b1e/out" "-L" "/tmp/cargo-installEL3oso/release/build/tree-sitter-e87a068496d3ec36/out" "-L" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "-ltree-sitter-lua-cpp" "-ltree-sitter-python-cpp" "-ltree-sitter-bash-cpp" "-ltree-sitter-ocaml-cpp" "-ltree-sitter-php-cpp" "-ltree-sitter-tsx" "-ltree-sitter-java" "-ltree-sitter-bash" "-ltree-sitter-lua" "-ltree-sitter-python" "-ltree-sitter-typescript" "-ltree-sitter-javascript" "-ltree-sitter-nix" "-ltree-sitter-php" "-ltree-sitter-json" "-ltree-sitter-c" "-ltree-sitter-ocaml-interface-cpp" "-ltree-sitter-ocaml" "-ltree-sitter-clojure" "-ltree-sitter-ruby-cpp" "-ltree-sitter-rust" "-ltree-sitter-cpp-cpp" "-ltree-sitter-css" "-ltree-sitter-elixir-cpp" "-ltree-sitter-commonlisp" "-ltree-sitter-dart" "-ltree-sitter-elisp" "-ltree-sitter-cpp" "-ltree-sitter-scala" "-ltree-sitter-ocaml-interface" "-ltree-sitter-ruby" "-ltree-sitter-janet-simple" "-ltree-sitter-go" "-ltree-sitter-c-sharp" "-ltree-sitter-elixir" "-ltree-sitter-haskell-cpp" "-Wl,-Bdynamic" "-lstdc++" "-Wl,-Bstatic" "-ltree-sitter-haskell" "/tmp/cargo-installEL3oso/release/deps/libterm_size-8c49d9042352db79.rlib" "/tmp/cargo-installEL3oso/release/deps/libwu_diff-336eb15992ccab2a.rlib" "/tmp/cargo-installEL3oso/release/deps/libwalkdir-8d4b4e58cd316feb.rlib" "/tmp/cargo-installEL3oso/release/deps/libsame_file-7765d8a3b595db26.rlib" "/tmp/cargo-installEL3oso/release/deps/libmimalloc-17656fb7fb48803f.rlib" "/tmp/cargo-installEL3oso/release/deps/liblibmimalloc_sys-34901f398ad9ab43.rlib" "/tmp/cargo-installEL3oso/release/deps/libtree_sitter-e7713f19aec42b0d.rlib" "/tmp/cargo-installEL3oso/release/deps/libtyped_arena-7746787a291b37c5.rlib" "/tmp/cargo-installEL3oso/release/deps/libconst_format-79862a20276d0963.rlib" "/tmp/cargo-installEL3oso/release/deps/libclap-d2ad5e1094c8d6d4.rlib" "/tmp/cargo-installEL3oso/release/deps/libtextwrap-4ca68cb450259d96.rlib" "/tmp/cargo-installEL3oso/release/deps/libindexmap-a08c394e3617c104.rlib" "/tmp/cargo-installEL3oso/release/deps/libhashbrown-ba8b0286cc482e62.rlib" "/tmp/cargo-installEL3oso/release/deps/libos_str_bytes-79f6343ccb1aead7.rlib" "/tmp/cargo-installEL3oso/release/deps/libbitflags-ab97cbe7c4be6312.rlib" "/tmp/cargo-installEL3oso/release/deps/libowo_colors-ddac0642a5f3aa29.rlib" "/tmp/cargo-installEL3oso/release/deps/liblazy_static-2929070d3a17e85e.rlib" "/tmp/cargo-installEL3oso/release/deps/libstrsim-d45870c496004e81.rlib" "/tmp/cargo-installEL3oso/release/deps/librpds-520589c5ede6dab2.rlib" "/tmp/cargo-installEL3oso/release/deps/libarchery-7f374d9050ae4d90.rlib" "/tmp/cargo-installEL3oso/release/deps/libradix_heap-b406031c2b7030fb.rlib" "/tmp/cargo-installEL3oso/release/deps/libitertools-7d3aa88a9e50bfb2.rlib" "/tmp/cargo-installEL3oso/release/deps/libeither-823745033d8b50ba.rlib" "/tmp/cargo-installEL3oso/release/deps/librustc_hash-a4fc922f683742fa.rlib" "/tmp/cargo-installEL3oso/release/deps/libpretty_env_logger-7c881f3cf7348cce.rlib" "/tmp/cargo-installEL3oso/release/deps/libenv_logger-58c0df83b9b29f4b.rlib" "/tmp/cargo-installEL3oso/release/deps/libtermcolor-6266713a5344c705.rlib" "/tmp/cargo-installEL3oso/release/deps/libatty-2749fa4a490326df.rlib" "/tmp/cargo-installEL3oso/release/deps/liblibc-eda48fb5bc11b482.rlib" "/tmp/cargo-installEL3oso/release/deps/libhumantime-c2de066a29d80cf9.rlib" "/tmp/cargo-installEL3oso/release/deps/libquick_error-98db65e70a8e3b45.rlib" "/tmp/cargo-installEL3oso/release/deps/libregex-d18b43748e6bfce6.rlib" "/tmp/cargo-installEL3oso/release/deps/libaho_corasick-4e07ff678d8318b3.rlib" "/tmp/cargo-installEL3oso/release/deps/libmemchr-bf813c5ed322d98a.rlib" "/tmp/cargo-installEL3oso/release/deps/libregex_syntax-a20091ecdfe0e599.rlib" "/tmp/cargo-installEL3oso/release/deps/liblog-43021d8e9c3a2e2b.rlib" "/tmp/cargo-installEL3oso/release/deps/libcfg_if-af4ed11818f18246.rlib" "-Wl,--start-group" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-443e20b28eeed844.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-3e337470e099b116.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libobject-10a87791239bd676.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libmemchr-0e6cad6dd623d38e.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libaddr2line-9d05dd05a4d51b33.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgimli-a60ff138fe1adfad.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_demangle-0e9344458c09713e.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd_detect-939385b13e387309.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libhashbrown-e3ce420901586b53.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libminiz_oxide-546bece8dd1bea42.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libadler-9b358af16d7d7aa1.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_alloc-554cd499fe1a3b52.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-fbad2f5de5fb5cb2.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcfg_if-8fec8a74c706e4b4.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-020914c5936c5f85.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-4896c4057dc9553a.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_core-2a6a2797f7a73818.rlib" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-0e3656b1fda5fd7b.rlib" "-Wl,--end-group" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-1f103368fa522bc0.rlib" "-Wl,-Bdynamic" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-L" "/home/joao/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/tmp/cargo-installEL3oso/release/deps/difft-17b0e132d3b60361" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro,-znow" "-Wl,-O1" "-nodefaultlibs"
  = note: /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-lua.a(parser.o):(.data.rel.ro.language.0+0xb8): undefined reference to `tree_sitter_lua_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-lua.a(parser.o):(.data.rel.ro.language.0+0xc0): undefined reference to `tree_sitter_lua_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-lua.a(parser.o):(.data.rel.ro.language.0+0xc8): undefined reference to `tree_sitter_lua_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-lua.a(parser.o):(.data.rel.ro.language.0+0xd0): undefined reference to `tree_sitter_lua_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-lua.a(parser.o):(.data.rel.ro.language.0+0xd8): undefined reference to `tree_sitter_lua_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-python.a(parser.o):(.data.rel.ro.language.0+0xb8): undefined reference to `tree_sitter_python_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-python.a(parser.o):(.data.rel.ro.language.0+0xc0): undefined reference to `tree_sitter_python_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-python.a(parser.o):(.data.rel.ro.language.0+0xc8): undefined reference to `tree_sitter_python_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-python.a(parser.o):(.data.rel.ro.language.0+0xd0): undefined reference to `tree_sitter_python_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-python.a(parser.o):(.data.rel.ro.language.0+0xd8): undefined reference to `tree_sitter_python_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-php.a(parser.o):(.data.rel.ro.language.0+0xb8): undefined reference to `tree_sitter_php_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-php.a(parser.o):(.data.rel.ro.language.0+0xc0): undefined reference to `tree_sitter_php_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-php.a(parser.o):(.data.rel.ro.language.0+0xc8): undefined reference to `tree_sitter_php_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-php.a(parser.o):(.data.rel.ro.language.0+0xd0): undefined reference to `tree_sitter_php_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-php.a(parser.o):(.data.rel.ro.language.0+0xd8): undefined reference to `tree_sitter_php_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-haskell.a(parser.o):(.data.rel.ro.language.0+0xb8): undefined reference to `tree_sitter_haskell_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-haskell.a(parser.o):(.data.rel.ro.language.0+0xc0): undefined reference to `tree_sitter_haskell_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-haskell.a(parser.o):(.data.rel.ro.language.0+0xc8): undefined reference to `tree_sitter_haskell_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-haskell.a(parser.o):(.data.rel.ro.language.0+0xd0): undefined reference to `tree_sitter_haskell_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-haskell.a(parser.o):(.data.rel.ro.language.0+0xd8): undefined reference to `tree_sitter_haskell_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-bash.a(parser.o):(.data.rel.ro.language.0+0xb8): undefined reference to `tree_sitter_bash_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-bash.a(parser.o):(.data.rel.ro.language.0+0xc0): undefined reference to `tree_sitter_bash_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-bash.a(parser.o):(.data.rel.ro.language.0+0xc8): undefined reference to `tree_sitter_bash_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-bash.a(parser.o):(.data.rel.ro.language.0+0xd0): undefined reference to `tree_sitter_bash_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-bash.a(parser.o):(.data.rel.ro.language.0+0xd8): undefined reference to `tree_sitter_bash_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml.a(parser.o):(.data.rel.language.0+0xb8): undefined reference to `tree_sitter_ocaml_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml.a(parser.o):(.data.rel.language.0+0xc0): undefined reference to `tree_sitter_ocaml_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml.a(parser.o):(.data.rel.language.0+0xc8): undefined reference to `tree_sitter_ocaml_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml.a(parser.o):(.data.rel.language.0+0xd0): undefined reference to `tree_sitter_ocaml_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml.a(parser.o):(.data.rel.language.0+0xd8): undefined reference to `tree_sitter_ocaml_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-cpp.a(parser.o):(.data.rel.ro.language.0+0xb8): undefined reference to `tree_sitter_cpp_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-cpp.a(parser.o):(.data.rel.ro.language.0+0xc0): undefined reference to `tree_sitter_cpp_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-cpp.a(parser.o):(.data.rel.ro.language.0+0xc8): undefined reference to `tree_sitter_cpp_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-cpp.a(parser.o):(.data.rel.ro.language.0+0xd0): undefined reference to `tree_sitter_cpp_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-cpp.a(parser.o):(.data.rel.ro.language.0+0xd8): undefined reference to `tree_sitter_cpp_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml-interface.a(parser.o):(.data.rel.language.0+0xb8): undefined reference to `tree_sitter_ocaml_interface_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml-interface.a(parser.o):(.data.rel.language.0+0xc0): undefined reference to `tree_sitter_ocaml_interface_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml-interface.a(parser.o):(.data.rel.language.0+0xc8): undefined reference to `tree_sitter_ocaml_interface_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml-interface.a(parser.o):(.data.rel.language.0+0xd0): undefined reference to `tree_sitter_ocaml_interface_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ocaml-interface.a(parser.o):(.data.rel.language.0+0xd8): undefined reference to `tree_sitter_ocaml_interface_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ruby.a(parser.o):(.data.rel.ro.language.0+0xb8): undefined reference to `tree_sitter_ruby_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ruby.a(parser.o):(.data.rel.ro.language.0+0xc0): undefined reference to `tree_sitter_ruby_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ruby.a(parser.o):(.data.rel.ro.language.0+0xc8): undefined reference to `tree_sitter_ruby_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ruby.a(parser.o):(.data.rel.ro.language.0+0xd0): undefined reference to `tree_sitter_ruby_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-ruby.a(parser.o):(.data.rel.ro.language.0+0xd8): undefined reference to `tree_sitter_ruby_external_scanner_deserialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-elixir.a(parser.o):(.data.rel.ro.language.0+0xb8): undefined reference to `tree_sitter_elixir_external_scanner_create'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-elixir.a(parser.o):(.data.rel.ro.language.0+0xc0): undefined reference to `tree_sitter_elixir_external_scanner_destroy'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-elixir.a(parser.o):(.data.rel.ro.language.0+0xc8): undefined reference to `tree_sitter_elixir_external_scanner_scan'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-elixir.a(parser.o):(.data.rel.ro.language.0+0xd0): undefined reference to `tree_sitter_elixir_external_scanner_serialize'
          /usr/bin/ld: /tmp/cargo-installEL3oso/release/build/difftastic-19c6e30c11924d38/out/libtree-sitter-elixir.a(parser.o):(.data.rel.ro.language.0+0xd8): undefined reference to `tree_sitter_elixir_external_scanner_deserialize'
          collect2: error: ld returned 1 exit status

  = help: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-link-libkindname)

error: could not compile `difftastic` due to previous error
error: failed to compile `difftastic v0.25.0`, intermediate artifacts can be found at `/tmp/cargo-installEL3oso`

  ```
</details>

## Workaround:

I cloned the repository and ran `cargo install --path .` it worked fine.

**Comment by Wilfred** [comment]
I think this was fixed in 42884b73a0376a27c75ac9c83608a77dbcae5af5 :)

**Comment by marcospb19** [comment]
Just to confirm, it was fixed, thanks!
